pkg/logflags/logflags.go

```diff
@@ -151,6 +171,8 @@ func Setup(logFlag bool, logstr string, logDest string) error {
 			debugLineErrors = true
 		case "rpc":
 			rpc = true
+		case "dap":
```

Comment:
The values that we recognize here are documented inside cmd/dlv/cmds/commands.go (grep for "Help about logging flags"). Please add it there too.